mir_perception/mir_object_recognition/ros/scripts/rgb_object_recognizer_node

```diff
@@ -178,11 +136,22 @@ if __name__ == '__main__':
     classifier_name = rospy.get_param("~classifier")
     dataset = rospy.get_param("~dataset")
     model_dir = rospy.get_param("~model_dir")
-    weights = os.path.join(roslib.packages.get_pkg_dir("mir_rgb_object_recognition_models"),
-                           'common', 'models', classifier_name, dataset, "best.pt")
-
+    model_category = rospy.get_param("~model_category")
+    weight_1 = os.path.join(roslib.packages.get_pkg_dir("mir_rgb_object_recognition_models"),
```

Comment:
instead of assuming there are two models, you should loop through and add all. You're anyway passing a list to RGBObjectRecognizer